debian/changelog

```diff
@@ -1,6 +1,7 @@
 ubuntu-advantage-tools (19.5) UNRELEASED; urgency=medium
 
   * New upstream release (LP: #1832757):
+    - livepatch: no auto-enable on attach for trusty
```

Comment:
you should update the contradicting entry further down, which says livepatch is autoenabled

Reply:
I made the other one say 'auto-enable on attach where supported'. In esm case anyway it's limited to trusty anyway. so they don't really conflict. This other is just more specific